### PR TITLE
80081 add file overview functionality to 7959c

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR adds the missing file overview screens to form 10-7959c. Two new pages have been added and the confirmation page has been updated. This feature was already implemented for form 10-10d, so this is mainly just importing things and setting up pass-through pages.

- Added missing file overview page
- Added missing file consent page
- Updated confirmation page to show missing files after submission.
- Team: IVC CHAMPVA
- Flipper: NA

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/80081

## Testing done

- Manual testing
- Verified unit tests run and pass

## Screenshots

|         | Screens | 
| ------- | ------ | 
| Missing files overview |Screenshot 2024-04-19 at 11 50 24|
| Consent screen |Screenshot 2024-04-19 at 12 03 47|
| Confirmation page |Screenshot 2024-04-19 at 11 55 20|

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by michaelclement** [comment]
This PR relies on these preceding PRs, so they need to merge first and then this will reduce in size significantly:
- https://github.com/department-of-veterans-affairs/vets-website/pull/29277
- https://github.com/department-of-veterans-affairs/vets-website/pull/29294
- https://github.com/department-of-veterans-affairs/vets-website/pull/29295

File: src/applications/ivc-champva/10-7959C/components/MissingFileConsentPage.jsx
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/components/MissingFileConsentPage.jsx:1]: This page shows a list of missing files and has a checkbox stating the user consents to mail the missing documents in at a later date.

File: src/applications/ivc-champva/10-7959C/containers/ConfirmationPage.jsx
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/containers/ConfirmationPage.jsx:1]: Updated the confirmation page that shows after submission - it now displays any missing files the user didn't upload as well as instructions on mailing/faxing them in.

**Review by reiting — APPROVED** [review]

**Review by balexandr — APPROVED** [review]
Lgtm

**Review by cloudmagic80 — APPROVED** [review]

**Review by powellkerry — APPROVED** [review]

**Pull request merged**